cmd/algocfg/profileCommand.go

```diff
@@ -91,13 +91,69 @@ var (
 		},
 	}
 
+	hybridRelay = configUpdater{
+		description: "Relay consensus messages across both ws and p2p networks, also support recent catchup.",
+		updateFunc: func(cfg config.Local) config.Local {
+			// WS relay config defaults
+			cfg.MaxBlockHistoryLookback = 22000 // Enough to support 2 catchpoints with some wiggle room for nodes to catch up from the older one
+			cfg.CatchpointFileHistoryLength = 3
+			cfg.CatchpointTracking = 2
+			cfg.EnableLedgerService = true
+			cfg.EnableBlockService = true
+			cfg.NetAddress = ":4160"
+			// This should be set to the public address of the node if public access is desired
+			cfg.PublicAddress = "PLEASE_SET_ME"
```

Comment:
The previous configs work out of the box, but this requires an explicit change before being functional. I guess we should include it since we need it for hybrid, but it feels broken somehow.

Reply:
What would you suggest as an alternative? I agree it's not great, but we can't generate the proper publicAddress, and we consider it required...